tools/trap/trap.c

```diff
@@ -367,7 +367,7 @@ int main(int argc, char *argv[])
 	char *dev_file;
 	char tty_path[TTYPATH_LEN] = {0, };
 	char tty_type[TTYTYPE_LEN] = {0, };
-	int choice;
+	char choice;
```

Comment:
Why did you change the int to char? is it needed?

Reply:
Minor change to facilitate accepting 'r' and 'x' as inputs for resetting and exiting.